### ISSUE TITLE
jaws project create failing on Region selection

### ISSUE THREAD

**Issue by IanMeyers** [serverless/serverless#298]
When using jaws project create (node 0.12.4 on macos), everything seems fine until it tries to load region names:

JAWS: Select a region for your project:
  > us-east-1
    us-west-2
    eu-west-1
    ap-northeast-1
JAWS: Creating CloudFormation Stack for your new project (~5 mins)...

Then it hangs - if you hit enter, it will proceed (using us-east-1). I am unable to create a project successfully in any region as it's now failing to create stacks.

**Comment by IanMeyers** [comment]
Further info - when it hangs, if you do anything other than hit 'Return' then you get:

/usr/local/lib/node_modules/jaws-framework/lib/utils/cli.js:231
    } else if (key.name == 'up' && Select.state.index > 1) {
                  ^
TypeError: Cannot read property 'name' of undefined
    at ReadStream.keypressHandler (/usr/local/lib/node_modules/jaws-framework/lib/utils/cli.js:231:19)
    at ReadStream.emit (events.js:110:17)
    at readline.js:1175:14
    at Array.forEach (native)
    at emitKeys (readline.js:993:10)
    at ReadStream.onData (readline.js:910:14)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:538:20)

**Comment by ac360** [comment]
Thanks @IanMeyers.  We have a big release coming.  I think this should be fixed in that.  Sorry for the inconvenience :)
